go/enclave/components/batch_registry.go

```diff
@@ -167,41 +156,11 @@ func (br *batchRegistryImpl) notifySubscriber(batch *core.Batch, isHeadBatch boo
 		*subscriptionChan <- batch
 	}
 
-	if br.eventSubscription != nil && isHeadBatch {
+	if br.eventSubscription != nil {
```

Comment:
Won't this change result in sending events multiple times ( for example when doing duplicate batches )

Reply:
that's ok. Most apps expect duplicate events on reorgs